plugins/CoreAdminHome/tests/Integration/Commands/ResetInvalidationsTest.php

```diff
@@ -272,6 +272,264 @@ public function getDryRunTestData(): iterable
+
+        $expectedOutput = '1 invalidations found:
++------+--------+--------+------------+------------+--------+---------------------+---------------------+-----------------+------------+
+| name | idsite | report | date1      | date2      | period | ts_invalidated      | ts_started          | processing_host | process_id |
++------+--------+--------+------------+------------+--------+---------------------+---------------------+-----------------+------------+
+| done | 1      |        | 2025-01-01 | 2025-01-01 | 1      | 2025-01-01 16:00:00 | 2025-02-05 11:30:00 | host1           | 1256       |
++------+--------+--------+------------+------------+--------+---------------------+---------------------+-----------------+------------+
+';
+
+        self::assertStringContainsString($expectedOutput, $this->applicationTester->getDisplay(true));
+
+        $code = $this->applicationTester->run([
+            'command' => 'core:reset-invalidations',
+            '--processing-host' => ['host1'],
+            '-vvv' => true,
+        ]);
+
+        self::assertEquals(1, $this->getInProgressInvalidationCount());
+        self::assertEquals(0, $code, $this->getCommandDisplayOutputErrorMessage());
+
+        $rows = Db::fetchAll('SELECT * FROM ' . Common::prefixTable('archive_invalidations'));
+
+        self::assertEquals(2, count($rows));
+
+        self::assertStringNotContainsString('Found duplicate invalidation', $this->getLogOutput());
+    }
+
+    public function testDuplicateInvalidationIsRemovedWhenBothAreReset()
+    {
+        Db::exec('TRUNCATE TABLE ' . Common::prefixTable('archive_invalidations'));
+
+        $invalidationsToInsert = [
+            [
+                'idarchive' => 7, 'name' => 'done', 'idsite' => 1, 'date1' => '2025-01-01', 'date2' => '2025-01-01',
+                'period' => Day::PERIOD_ID, 'ts_invalidated' => '2025-01-01 16:00:00', 'status' => ArchiveInvalidator::INVALIDATION_STATUS_IN_PROGRESS,
+                'report' => null, 'ts_started' => '2025-02-05 11:30:00', 'processing_host' => 'host1', 'process_id' => 1256
+            ],
+            [
+                'idarchive' => null, 'name' => 'done', 'idsite' => 1, 'date1' => '2025-01-01', 'date2' => '2025-01-01',
+                'period' => Day::PERIOD_ID, 'ts_invalidated' => '2025-01-03 02:55:55', 'status' => ArchiveInvalidator::INVALIDATION_STATUS_IN_PROGRESS,
+                'report' => null, 'ts_started' => '2025-01-01 08:00:00', 'processing_host' => 'host4', 'process_id' => 22555
+            ],
+        ];
+
+        $sql = 'INSERT INTO ' . Common::prefixTable('archive_invalidations')
+            . ' (`idarchive`, `name`, `idsite`, `date1`, `date2`, `period`, `ts_invalidated`, `status`, `report`, `ts_started`, `processing_host`, `process_id`)'
+            . ' VALUES (?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?)';
+
+        foreach ($invalidationsToInsert as $invalidation) {
+            Db::query($sql, $invalidation);
```

Comment:
```suggestion
            Db::query($sql, array_values($invalidation));
```

Because we can't have nice things at the moment:

> ArgumentCountError: mysqli_stmt::bind_param() does not accept unknown named parameters

Should we move all those inserts into something like `$this->insertInvalidations($invalidationsToInsert)`? Otherwise there are more occurrences that require the `array_values` workaround (so we can keep "readable" invalidation arrays) for php8-mysqli.